doc/model_details/noise_generator.ipynb

```diff
@@ -267,7 +267,7 @@
    "outputs": [],
    "source": [
     "def noise_params(V_mean, V_std, dt=1.0, tau_m=10., C_m=250.):\n",
-    "    'Returns mean and std for noise generator for parameters provided; defaults for iaf_neuron.'\n",
+    "    'Returns mean and std for noise generator for parameters provided.'\n",
```

Comment:
Maybe leaving the explanation about the fact that ``tau_m=10., C_m=250.`` are the default values for the ``iaf_psc_alpha`` model could be useful...